docs/src/modules/java/pages/actions.adoc

```diff
@@ -130,8 +130,43 @@ Scala::
 include::example$scala-fibonacci-action/src/main/scala/com/example/fibonacci/Main.scala[]
 ----
 
-By default, the generated constructor has a `ActionCreationContext` parameter, but you can change this to accept other parameters. 
+By default, the generated constructor has an `ActionCreationContext` parameter, but you can change this to accept other parameters. 
 If you change the constructor of the `FibonacciAction` class you will see a compilation error here, and you have to adjust the 
 factory function that is passed to `AkkaServerlessFactory.withComponents`.
 
 When more components are added the `AkkaServerlessFactory` is regenerated, and you have to adjust the registration from the `Main` class.
+
+== Testing the Action 
+
+
+=== Unit tests
+
+The following snippet shows how the `CounterJournalToTopicActionTestKit` is used to test the `CounterJournalToTopicAction` implementation.
+
+AkkaServerless provides the `CounterJournalToTopicActionTestKit` that allows us to call the methods of `CounterJournalToTopicAction`. For each `Action` AkkaServerless generates an specific test kit for it, with the name `[ActionName]TestKit`. This test kit hold an instance of its `Action`. Each call we execute over to the test kit returns an `ActionResult` that holds the effect produced by the underlying `CounterJournalToTopicAction`. 
+
+Apart from the test kit, to facilitate the creation of unit tests, AkkaServerless generates test classes, ready to use, based on the Action defined in the .proto files. This is shown in the snippet below. 
```

Comment:
```suggestion
Apart from the test kit AkkaServerless generates test classes based on the Action defined in the .proto files. This is shown in the snippet below. 
```

We can have a more straightforward text if we remove the `, something in the middle,`. In that case here, it's clear that we generate it to facilitate the creation of tests and that the generated tests is ready for use.